.buildbot.sh

```diff
@@ -14,18 +14,22 @@ cmake -DCMAKE_INSTALL_PREFIX=${INST_DIR} \
     ../llvm
 make -j `nproc` install
 
+# clang-format any new files that we've introduced ourselves.
+#
+# This requires llvm release tags to be consistent llvm upstream.
+PATH=${INST_DIR}/bin:${PATH}
+git remote add llvm https://github.com/llvm/llvm-project
+git fetch --tags llvm
+sh yk_format_new_files.sh
+git diff --exit-code
```

Comment:
Since this file is `set -e`, won't this line cause the whole buildbot execution to fail?

Reply:
That's the plan yes. If `clang-format` changed any files, then the formatting is wrong and we don't want to continue.